tap_azure_git/__init__.py

```diff
@@ -1049,7 +1049,7 @@ def get_all_pipelines(schema, org, repo_path, state, mdata, start_date):
                 pipeline = {
                     **raw_pipeline,
                     '_sdc_repository': sdc_repository,
-                    '_sdc_id': '{}/pipeline/{}/{}'.format(sdc_repository, raw_pipeline['id'], raw_pipeline['revision'])
+                    '_sdc_id': '{}/pipeline/{}'.format(sdc_repository, raw_pipeline['id'])
```

Comment:
This change will make pipelines only have the current revision.  We probably aren't looking at old revisions, but this will overwrite the existing pipeline if a new revision comes in.

Reply:
Yeah, I think that is what we want. If we want to get into looking at revisions of pipelines, then we'll need to pull in the information about revisions. For the purposes of what we're doing, revision X and revision Y of the same pipeline should be considered the same pipeline.